pkg/cmd/repo/autolink/view/view_test.go

```diff
@@ -91,17 +90,18 @@ func TestNewCmdView(t *testing.T) {
 			} else {
 				require.NoError(t, err)
 				assert.Equal(t, tt.wantExporter, gotOpts.Exporter != nil)
+				assert.Equal(t, tt.output.ID, gotOpts.ID)
```

Comment:
This test should always have been checking that the opts including the correct `ID`.